locales/es.json

```diff
@@ -94,5 +94,7 @@
     "darkModeToggle": "Cambiar a modo oscuro",
     "buttons": {
         "ok": "Vale"
-    }
+    },
+    "unableGetServerConfiguration": "Error al obtener la configuración del servidor",
+    "unableGetPublicUsers": "Error al obtener los usuarios"
```

Comment:
Shouldn't this be done through weblate to reduces the issues of concflicts

Reply:
@Camc314 Perhaps, I just did it here as I saw some other PRs with added strings in their own language, and in web we did the same stuff.

We should reach a consensus on this (at least between the main contributors) so we know how to review future PRs in this matter and not do it anymore. cc @MrTimscampi

Reply:
This sounds good to me.

Reply:
I think we should probably avoid changing any strings other than `en-US`, unless we're re-organizing strings (Which I want to do at some point since we know have confirmation that nested strings work well with Weblate).

This avoids cases where we unintentionally break Weblate, leaving our translators with old strings until one of us notices it.